website/content/en/docs/getting-started/_index.md

```diff
@@ -24,3 +24,5 @@ Learn more about Karpenter and how to get started below.
 * [EC2 Spot Workshop for Karpenter](https://ec2spotworkshops.com/karpenter.html)
 * [EKS Karpenter Workshop](https://www.eksworkshop.com/docs/autoscaling/compute/karpenter/)
 * [Advanced EKS Immersion Karpenter Workshop](https://catalog.workshops.aws/eks-advanced/karpenter/)
+* [Karpenter Blueprints](https://github.com/aws-samples/karpenter-blueprints)
+* [Tutorial: Run Kubernetes Clusters for Less with Amazon EC2 Spot and Karpenter](https://community.aws/tutorials/run-kubernetes-clusters-for-less-with-amazon-ec2-spot-and-karpenter#step-6-optional-simulate-spot-interruption)
```

Comment:
Can we make sure this change is propagated across all the docs? We take the preview docs forward so we at least need to make this change there.

Reply:
@jonathan-innis done :)